.travis.yml

```diff
@@ -17,4 +17,9 @@ after_success:
   - mkdir _site public
   - python manage.py collectstatic --noinput
   - python manage.py distill-local public --force
+  - mkdir public/activity
+  - python activity/scraper.py
```

Comment:
Move all of this before the django stuff.
Then the json could be loaded by djanjo, but more importantly the static components should all be managed by djanjo's asset system. (e.g. there are django addons that do cache busting)